ner_v2/tests/temporal/time/time_ner_tests.yaml

```diff
@@ -0,0 +1,552 @@
+args:
+    timezone: "UTC"
```

Comment:
I am not sure on this but we should probably use `Asia/Kolkata` if our test writers are going to be mostly in India

Reply:
Okay